otx/mpa/modules/models/heads/custom_atss_head.py

```diff
@@ -168,18 +168,16 @@ def loss_single(
             pos_centerness = centerness[pos_inds]
 
             centerness_targets = self.centerness_target(pos_anchors, pos_bbox_targets)
-            pos_decode_bbox_pred = self.bbox_coder.decode(pos_anchors, pos_bbox_pred)
-            pos_decode_bbox_targets = self.bbox_coder.decode(pos_anchors, pos_bbox_targets)
+            if self.reg_decoded_bbox:
+                pos_bbox_pred = self.bbox_coder.decode(pos_anchors, pos_bbox_pred)
 
             if self.use_qfl:
-                quality[pos_inds] = bbox_overlaps(
-                    pos_decode_bbox_pred.detach(), pos_decode_bbox_targets, is_aligned=True
-                ).clamp(min=1e-6)
+                quality[pos_inds] = bbox_overlaps(pos_bbox_pred.detach(), pos_bbox_targets, is_aligned=True).clamp(
+                    min=1e-6
+                )
```

Comment:
In my opinion, `reg_decoded_bbox` means that the model should output the decoded bbox directly. So, the `pos_bbox_targets` should be decoded to enforce model to produce decoded output from its regression head.

Reply:
Whether the final outputs of model are decoded or not is handled in simple_test funciton. Since it is loss calculation funciton, I think the only thing to check is whether the predictions and targets are in same scale. 
If you're worrying is that the loss do not reflect the actual value of the model's prediction, please let me know

Reply:
If it's in simple_test(), then it's alright.
Thank you for the fix.

Reply:
![image](https://user-images.githubusercontent.com/106788349/224929583-077887ef-4ed6-41d5-ac99-3fa0f1549932.png)
For sure, it's in get_bboxes function in base_dense_head.py